### PR TITLE
Registering runtimes in Compass Director

### PR DESCRIPTION
**Pull request by mvshao**
**Description**

Compass Manager now is able to register runtimes in Compass Director

**Changes proposed in this pull request:**

- registered runtimes in the Compass Director

- changed the version of Kyma Resource API to v1beta2, we are using the latest version

- ginkgo tests adapted

**Related issue(s)**
#10

### CODE REVIEW

**Review by akgalwas — COMMENTED** [review]
It looks like a good base for next steps with Compass Manager, however, we need to adjust to the new requirements.

File: controllers/registrator.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	}
+	return string(runes)
+}
+
+func createRuntimeLabels(kymaLabels map[string]string) map[string]interface{} {
+
+	runtimeLabels := make(map[string]interface{})
+
+	// check if label exists?
+	runtimeLabels["kyma_managed_by"] = kymaLabels[ManagedByLabel]
```
[the remark this answers is not in the document]
Comment (mvshao) [reply controllers/registrator.go:81]: Yes, you are right about it. During implementation, I was convinced that this information could help in resolving incidents. But now I agree with you this information doesn't have value in the Compass context.

**Pull request merged**